File: 2017/solutions/prac02.java
import java.io.*;
 
public class prac02 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader
            = new BufferedReader(new InputStreamReader(System.in));
 
        int numCases = Integer.parseInt(reader.readLine());


        while (numCases > 0) {
            --numCases;
            String line[] = reader.readLine().trim().split("\\s+");
            String noun = line[0];
            String verb = line[1];
            System.out.println(noun + " is " + verb + " today!");
        }
    }
}

File: 2017/solutions/prac02.cpp
#include <iostream>
#include <string>
 
using namespace std;
 
int main(int argc, char **argv) {
    int numCases;
    string name, verb;
    cin >> numCases;
    while (numCases) {
        --numCases;
        cin >> name >> verb;
        cout << name << " is " << verb << " today!" << endl;
    }
    return 0;
}

